delta/introduction.adoc

```diff
@@ -38,18 +51,18 @@ The response contains a <<{serialization}.adoc#SerializationChunk, serialization
 8. Client updates its editor or processing with the event's contents.
 9. Client <<qry-SignOff, ends>> the participation.
 
-The repository is always the ultimate arbiter of its contents.{fn-org339}
+The server is always the ultimate arbiter of its contents.{fn-org339}
 It can decide to apply each command verbatim, with some tweaks (e.g. because of concurrent changes), or not at all (e.g. because the command would introduce containment loops).
 
-Each repository might introduce its own strategy how to handle commands.{fn-org291}
+Each server might introduce its own strategy how to handle commands.{fn-org291}
 For example, assume client 1 sends a `deleteChild` command to delete node `X`, and then client 2 sends a `setProperty` command on node `X`.
 
-* Repository A applies all commands in order, thus deletes node `X` and ignores the second command.
+* Server A applies all commands in order, thus deletes node `X` and ignores the second command.
```

Comment:
I think what matters is that commands _pertaining to a certain repository_ are dealt in order. I could also process all commands pending for a certain repository, even if there are others older for an other repository